package.json

```diff
@@ -1,29 +1,43 @@
 {
-    "name": "brightlayer-ui-cli-templates-react",
-    "version": "0.0.0",
+    "name": "@brightlayer-ui/react-cli-templates",
+    "version": "1.0.0-alpha.0",
     "keywords": [
         "template",
         "brightlayer-ui",
         "brightlayer ui",
-        "cli"
+        "React",
+        "Vite",
+        "Typescript",
+        "Templates",
+        "React test library",
+        "Jest",
+        "Prettier",
+        "ESlint"
     ],
-    "description": "This repository holds the different react templates used by the Brightlayer UI CLI",
+    "description": "This repository holds the different react vite templates used by Brightlayer UI",
+    "main": "index.js",
+    "type": "module",
+    "bin": {
+        "create-blui-react-app": "./index.js"
+    },
     "repository": {
         "type": "git",
         "url": "https://github.com/etn-ccis/blui-react-cli-templates.git",
         "directory": "blank-typescript"
     },
     "license": "BSD-3-Clause",
     "engines": {
-        "node": ">=10"
+        "node": "^18.0.0 || >=20.0.0"
```

Comment:
What does this do? Can we just use 20+? Not sure why we specifically call out 18